src/VisualStudio/VisualBasic/Impl/Options/VisualBasicSettingsManagerOptionSerializer.vb

```diff
@@ -165,9 +167,26 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Options
                 Return FetchEnterKeyBehavior(optionKey, value)
             End If
 
+            If optionKey.Option Is CompletionOptions.TriggerOnDeletion Then
+                Return FetchTriggerOnDeletion(optionKey, value)
+            End If
+
             Return MyBase.TryFetch(optionKey, value)
         End Function
 
+        Private Function FetchTriggerOnDeletion(optionKey As OptionKey, ByRef value As Object) As Boolean
+            If MyBase.TryFetch(optionKey, value) Then
+                If value Is Nothing Then
+                    ' The default behavior for VB is to trigger completion on deletion.
+                    value = CType(True, Boolean?)
```

Comment:
Why are the values here nullable - you seem to always specify a value, never return null?